backend/api/tests/test_models.py

```
@@ -0,0 +1,156 @@
+    user = User.objects.create_user(email='company@example.com', firebase_uid='companyuid', role='company')
+    profile = CompanyProfile.objects.create(
+        user=user,
+        company_name='Test Corp',
+        company_address='123 Test St',
+        industry='Tech'
+    )
+    assert profile.user == user
+    assert profile.company_name == 'Test Corp'
+    assert str(profile) == 'Test Corp'
+
+def test_viewer_profile_creation():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_viewer_profile_creation`

Add return type annotation: `None`

(ANN201)